src/js/common/schemaform/actions.js

```diff
@@ -25,11 +27,12 @@ export function setEditMode(page, edit, index = null) {
   };
 }
 
-export function setSubmission(field, value) {
+export function setSubmission(field, value, extra = null) {
```

Comment:
This is where Flow would come in handy. What's `extra` for? Could you add some comments explaining why we might want `extra`?

I can see in this case we're using it to record the `x-ratelimit-reset` response header (right?), but it's potentially useful for anything we want to record from the api response, yeah?

Reply:
Yeah, I needed a way to pass info from the submission error into the state. Not a big fan of this, but couldn't think of a better way of doing it.